vmdb/app/services/orchestration_template_dialog_service.rb

```diff
@@ -0,0 +1,140 @@
+class OrchestrationTemplateDialogService
+  def create_dialog(dialog_label, template)
+    Dialog.new(:label => dialog_label, :buttons => "submit,cancel").tap do |dialog|
+      tab = dialog.dialog_tabs.build(:display => "edit", :label => "Basic Information", :position => 0)
+      add_stack_group(tab, 0)
+
+      template.parameter_groups.each_with_index do |parameter_group, index|
+        add_parameter_group(parameter_group, tab, index + 1)
+      end
+      dialog.save!
+    end
+  end
+
+  private
+
+  def add_stack_group(tab, position)
+    tab.dialog_groups.build(
+      :display    => "edit",
+      :label      => "Options",
+      :position   => position
```

Comment:
Remove the extra spaces between the key and hash rocket to get rid of rubocop warnings.